api/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:17-alpine
```

Comment:
This is the parent image
We look for it in docker hub this is a 17th verrsion of nodejs installed apline OS which is a short versino of Linux